cfpb/consumerfinance.gov
# Graduation Path Tool No JS Alert

## Changes

- Added alert to notify the user if their JS is not turned on. This will cover the case of using Internet Explorer as well

## How to test this PR

1. Run locally
2. [Go to path to grad tool](http://127.0.0.1:8000/paying-for-college/your-financial-path-to-graduation/?iped=176080&pid=1408-3&houp=onCampus&typp=bachelors&prop=0&lenp=4&ratp=inState&depp=dependent&cobs=n&utm_source=vanity&utm_medium=redirect&utm_campaign=gradpath&tuit=8650&hous=9815&book=1200&subl=2500&unsl=2000&houx=1265&fdx=537&clhx=131&trnx=858&hltx=328&entx=193&retx=393&taxx=344&othx=370)
3. Page should load. Now disable JavaScript and refresh

## Screenshots

<img width="1272" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/85513449/434b748f-4fa0-4565-8a68-e1ba3beeeb31">

Completes issue 97 on XP board.
